R/fotmob_leagues.R

```diff
@@ -51,7 +51,7 @@
   n_league_urls <- nrow(league_urls)
   if(n_league_urls == 0) {
     stop(
-      'Could not find any leagues matching specified parameters.',
```

Comment:
the trailing comma here was an issue. if the logic ended here, then the error message would be about the trailing comma, not `"Could not find any leagues matching specified parameters."`